testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -126,7 +207,7 @@ class DensityParameters(object):
                   'solute': "protein and not name H*",
                   }
     references = {'static':
-                      {'meandensity': 0.016764271713091212, },
+                  {'meandensity': 0.016764271713091212, },
```

Comment:
The indentation here is no longer consistent with the rest of the dictionary, unless there's a good reason, I'd just stick with what it was?

Reply:
Thanks for pointing this out, indentation was unintended. Fixed and lines below.